R/plotly.R

```diff
@@ -54,8 +46,8 @@
 #' 
 plot_ly <- function(data = data.frame(), ..., type = "scatter",
                     group, color, colors, symbol, symbols, size,
-                    filename, fileopt, world_readable = TRUE,
-                    inherit = TRUE, evaluate = FALSE) {
+                    width = NULL, height = NULL, inherit = TRUE, 
+                    evaluate = FALSE) {
```

Comment:
In the spirit of backwards compatability, we should probably issue a `warning` if they supplied `filename`, `fileopt`, or `world_readable` and tell them to use `plotly_POST`